TrafficCapture/README.md

```diff
@@ -34,9 +34,9 @@ server, recording the packet traffic of the new interactions.
 
 Learn more about its functionality and setup here: [Traffic Replayer](trafficReplayer/README.md)
 
-### OpenSearch Benchmark
+### Migration Console
 
-A container with a [script](dockerSolution/src/main/docker/openSearchBenchmark/runTestBenchmarks.sh) to run different [OpenSearch Benchmark](https://github.com/opensearch-project/opensearch-benchmark) workloads
+A container with a [script](dockerSolution/src/main/docker/migrationConsole/runTestBenchmarks.sh) to run different [OpenSearch Benchmark](https://github.com/opensearch-project/opensearch-benchmark) workloads
 is brought up as part of the solution.
```

Comment:
I don't think we have to add this right now, but I think it would be good to include an example of the copilot command to exec into this container.